Portal/ClientApp/sass/common/site-header.scss

```diff
@@ -0,0 +1,6 @@
+.govuk-header__language{
+    float: right;
```

Comment:
Are there any gov inbuilt styles that will push to link to the right that already exist?

Reply:
There is but it means we have to use 2 different classes. It's still better so I have changed.

<div class="govuk-header__navigation--end">
    <a href="?culture=@oppositeCultureValue" class="govuk-header__language">@oppositeCultureName</a>
    </div>
</div>